app/studio/1.0.x/deploy-to-dev-portal.md

```diff
@@ -4,29 +4,28 @@ title: Deploy to the Dev Portal
 
 ### Introduction
 
-By connecting to your Kong Enterprise instance, Kong Studio can sync Spec files to
-and from your Dev Portal.
-
+By connecting to your Kong Enterprise instance, Kong Studio's Insomnia Designer 
+Kong Plugin Bundle can sync Spec files to and from your Dev Portal.
 
 ### Prerequisites
 
+* [Insomnia Designer + Insomnia Designer Kong Plugin Bundle](/studio/{{page.kong_version}}/download-install)
 * Running Kong Enterprise 1.3 or later
 * Admin API privileges for the Workspace you will be connecting to
 * The Kong Developer Portal is enabled and running
 
-
-
 ### Step 1. Connect to Kong Enterprise
 
-From Kong Studio:
+From Insomnia Designer:
 
-* Open the spec you want to deploy to the Developer Portal
-* Click on the **caret menu** and select **Deploy to Kong Portal**
+* Go to the Documents Listing View
+* Find the spec you want to deploy to the Developer Portal
+* Click on the documents **action menu** and select **Deploy to Kong Portal**
 
-![Menu](https://doc-assets.konghq.com/studio/1.0/dev-portal/01-menu.png)
+![Menu](https://s3.amazonaws.com/helpscout.net/docs/assets/59e383122c7d3a40f0ed78e2/images/5ea7f9d02c7d3a7e9aebbe6e/file-ZA5DLrBBPs.png)
```

Comment:
Tab the image and the result (the prompt) to fall into the step.

Reply:
I wonder if this can also be done through the inline editor